src/libstd/io/mod.rs

```diff
@@ -250,7 +250,7 @@
 //! [`println!`]: ../macro.println.html
 //! [`Lines`]: struct.Lines.html
 //! [`io::Result`]: type.Result.html
-//! [`try!`]: ../macro.try.html
+//! [`?` operator]: ../../book/syntax-index.html
```

Comment:
its sort of a shame that one cannot link right to the `?` operator within the index, but AFAICT there is no anchor for it specifically. (You could link to https://doc.rust-lang.org/stable/book/syntax-index.html#operators-and-symbols but I'm not sure if that pays off.)